docroot/themes/custom/uids_base/uids_base.libraries.yml

```diff
@@ -354,6 +354,9 @@ node-type-article:
   css:
     component:
       assets/css/content/node--type--article.css: { preprocess: false }
+      assets/css/components/toggletip.css: { preprocess: false }
```

Comment:
Should this be a separate library to only load if caption/global caption is available (within the if check of uids_base.theme) or if this is used on other content types in the future?